terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
yhrun: error: cn9707: task 3: Aborted
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
